scss/_patterns_search-and-filter.scss

```diff
@@ -0,0 +1,168 @@
+@import 'settings';
+
+@mixin vf-p-search-and-filter {
+  @include vf-search-and-filter;
+  @include vf-filter-panel-section;
+}
+
+@mixin vf-search-and-filter {
+  .p-search-and-filter {
+    $height: 2.5rem;
+
+    border: 1px solid $color-mid-dark;
+    position: relative;
+
+    .p-search-and-filter__search-container {
+      align-items: center;
+      background-color: $color-x-light;
+      display: flex;
+      flex-wrap: wrap;
+      height: auto;
+      margin: 0;
+      min-height: $height;
+      overflow: hidden;
+      padding: 0 2rem 0 0.5rem;
+      position: relative;
+
+      &[data-active='true'] {
+        height: auto;
+      }
+
+      &[data-empty='false'],
+      &[aria-expanded='false'] {
+        height: $height;
+      }
+
+      .p-chip {
+        margin-bottom: 0.5rem;
+      }
+
+      .p-search-and-filter__selected-count {
+        color: $color-information;
+        cursor: pointer;
+        position: absolute;
+        right: 0.5rem;
+        top: 6px;
+      }
+
+      &[aria-expanded='true'] {
+        height: auto;
+        max-height: 100%;
+
+        .p-search-and-filter__selected-count {
+          display: none;
+        }
+      }
+
+      .p-search-box {
```

Comment:
We have a lot more overrides for `.p-search-box` in line 101.
Why is this one separated? Search box seems to be always in `p-search-and-fliter__search-container`, so there doesn't seem to be a need for different styling in different contexts.

Reply:
I have removed the dependency on the search box.